corpus/expressions.txt

```diff
@@ -378,7 +378,7 @@ yield from a
       (expression_list (identifier))
       (yield (expression_list (integer)))))
   (expression_statement
-    (yield (identifier))))
+    (yield (expression_statement (identifier)))))
```

Comment:
Yield statements can contain yield statements, and so on, so to simplify the grammar, I've updated `expression_statement` rules to consistently match against `expression_statement` rather than `_expression`. This adds an extra layer in certain cases in the tree, but gives us a more consistent shape that better reflects the grammar specification than before.